### ISSUE TITLE
compiler segfaults without stacktrace

### ISSUE THREAD

**Issue by timotheecour** [timotheecour/Nim#15]
compiler segfaults
```nim
type Foo[N: static int] = array[cint(0) .. cint(N), float] # segfault
# type Foo[N: static int] = array[int32(0) .. int32(N), float] # ok
type T = Foo[3]
```

**Comment by ringabout** [comment]
This is already fixed.

**Comment by timotheecour** [comment]
thanks; indeed, I'll keep this open, because we still need a regression test case

**Comment by ringabout** [comment]
This is fixed in Nim 1.0.0, I will add a testcase.

**Comment by timotheecour** [comment]
Thanks @xflywind !
